src/dwn/dwn-utils.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Uses duck typing to determine whether the stream is a web browser ReadableStream
+ * or a Node.js Readable stream.
+ */
+export function isReadableWebStream(stream) {
+  return typeof stream._read !== 'function';
```

Comment:
can we not do something like `globalThis.ReadableStream && stream instanceof globalThis.ReadableStream` instead?

Reply:
There is ongoing discussion with @mistermoe and @thehenrytsai as to how to handle streams in web/browser vs. Node environments.  This implementation is a temporary workaround until we make a decision on what type of stream (web vs node) to expose in dwn-sdk-js.  Potentially we can move to using the web ReadableStream everywhere but there is more validation to be done to confirm all of the encryption libraries we are using in `dwn-sdk-js` can work with web ReadableStream.

I'll leave this as is for now but open an issue to track making a decision on how to handle the different stream types.